package.json

```diff
@@ -29,7 +29,7 @@
         "react-dom": "17.0.2",
         "react-icons": "^4.3.1",
         "react-responsive": "^9.0.0-beta.4",
-        "swr": "^1.0.1"
+        "swr": "^1.1.2"
```

Comment:
While doing a major refactoring already could you update other deps as well? Dependabot has been crying about the used `next` for a while now